### PR TITLE
Enable reaction avatars for teams' discussions

### PR DESCRIPTION
**Pull request by lukaszklis**
I found out today the reaction avatars were not working for the recently launched ”Discussions” feature on GitHub. Unfortunately I do not have a live public example to link directly to test it, however I attach some screenshots before and after the change.

**Before:**
before

**After:**
after

### CODE REVIEW

File: source/libs/page-detect.js [hunk 1]
```diff
@@ -88,3 +88,5 @@ export const isSingleCommit = () => /^commit\/[0-9a-f]{5,40}/.test(getRepoPath()
 export const isSingleFile = () => /^blob\//.test(getRepoPath());
 
 export const isTrending = () => location.pathname.startsWith('/trending');
+
+export const isDiscussion = () => /^([^/]+[/][^/]+\/)teams[/][^/]+[^/]+($|[/]$|[/]discussions)/.test(getCleanPathname());
```
Comment (sindresorhus) [inline source/libs/page-detect.js]: `[/]` => `\/`

File: source/libs/page-detect.js [hunk 2]
```diff
@@ -88,3 +88,5 @@ export const isSingleCommit = () => /^commit\/[0-9a-f]{5,40}/.test(getRepoPath()
 export const isSingleFile = () => /^blob\//.test(getRepoPath());
 
 export const isTrending = () => location.pathname.startsWith('/trending');
+
+export const isDiscussion = () => /^([^/]+\/[^/]+\/)teams\/[^/]+[^/]+($|\/$|\/discussions)/.test(getCleanPathname());
```
Comment (bfred-it) [inline source/libs/page-detect.js]: I think `\/$|` isn't necessary. `getCleanPathname()` never ends with a trailing slash.

**Comment by lukaszklis** [comment]
🙈 of course! Will update the PR in a few. :)

**Comment by lukaszklis** [comment]
@bfred-it thanks!

**Comment by bfred-it** [comment]
Thanks @lukaszklis!

**Comment by lukaszklis** [comment]
@bfred-it A little offtopic question related to the changes requested in the PR: does it make sense to introduce linters whenever people contribute to the repo? It might save maintainers’ priceless time on trivial topics. :)

**Comment by bfred-it** [comment]
You mean [custom rules](https://eslint.org/docs/developer-guide/working-with-rules)? They'd be great, but then we'd have to also (know how to) code and debug them.

**Pull request merged**